wowchemy-cms/assets/js/cms-preview-templates/post.js

```diff
@@ -0,0 +1,28 @@
+let PostPreview = createClass({
```

Comment:
Creates a custom preview for posts in order to properly show the `featured.*` image.
See https://www.netlifycms.org/docs/customization/